### PR TITLE
Make it work :)

### PR DESCRIPTION
**Pull request by TheFes**
Hi

I've had a look at your work, and got it working.
You need to split up everything per device_class, similar as for the binary sensors.
And you need to use slots instead of requires context.

Because there can be more sensors per area, I've split them up as well, and created a response which returns an average.

I've created separate sensor files for these intents

### CODE REVIEW

**Comment by TheFes** [comment]
Oh, and I commented out the `climate_HAssClimateGetTempterature` intents, because they clash with these new sensor temperature ones.

File: sentences/en/_common.yaml [hunk 1]
```diff
@@ -285,12 +285,18 @@ lists:
         out: "closed"
 
   # sensors
-  sensor_classes:
+  sensor_temperature:
```
Comment (spuljko) [inline sentences/en/_common.yaml:288]: I can't understand why is this changed. My idea is to create logic like covers have - there is cover devices. That way fewer sentences are needed.

Comment (TheFes) [reply sentences/en/_common.yaml:288]: Because you need sentences per device class, like for the binary sensors.  I could not get it working otherwise.

**Comment by spuljko** [comment]
Thank you @TheFes! 
For multiple sensors in same are I think listing devices and values have more meaning, for example, there could be fridge sensor and climate temperature sensor, comparting those values or getting average doesn't make a sense.

File: sentences/en/_common.yaml (hunk 1 above)
Comment (TheFes) [reply sentences/en/_common.yaml:288]: But it could be I'm wrong here. But this is how I got a working solution.

**Comment by TheFes** [comment]
Yeah, I didn't think of that (the fridge).
I think temperature should get a double domain approach in the end (climate and sensor). 

There's an open issue on that

File: sentences/en/_common.yaml (hunk 1 above)
Comment (spuljko) [reply sentences/en/_common.yaml:288]: I was trying to create it like in covers ("(is|are) any {cover_classes:device_class} {cover_states:state} [in <area>]"). I can't figure out why that approach isn't working..

**Pull request merged**